### ISSUE TITLE
Cannot read property ‘prototype’ of undefined

### ISSUE THREAD

**Issue by mtjoernelund** [guidone/node-red-contrib-chatbot#370]
After a restart of NodeRED I suddenly get “TypeError: Cannot read property ‘prototype’ of undefined” for both chatbot-redbot, chatbot-telegram-receive and chatbot-slack-blocks. 

Prior to restart chatbot was working fine with my telegram bot. 

All software is up to date and I have tried uninstalling and reinstalling chatbot. Also tried searching for similar issues but have not found anything. 

I am running Node-RED v.1.0.3, Node v13.4.0 on a Raspberry Pi 3b running Raspbian stretch. 

Any help resolving this is greatly appreciated. Any additional information that I need to provide?

**Comment by guidone** [comment]
this is strange, did you recently update nodejs or reinstalled redbot?

**Comment by mtjoernelund** [comment]
Nope

**Comment by mtjoernelund** [comment]
After ensuring that all other nodes and everything else is completely updated, I just tried re-installing chatbot from the palette manager. Same result. I pops up telling me that all the chatbot nodes are installed, but when I look in the node-bar they are not there and the log throws the same TypeError.

Any help in deleting all of the flows that I have created that contain chatbot and try to reinstall after that? Would rather avoid that, as I would need to re-create all these flows.
